opa/populate_build_platform_vuln_database_test.go

```diff
@@ -0,0 +1,227 @@
+//go:build build_platform_vuln_database
+// +build build_platform_vuln_database
```

Comment:
Add instructions to `README.md` maybe on how to use this ? 
Or just here as a comment, since it's more a maintenance thing than user ?